pg/lib/dialect/node_postgres_driver_dialect.ts

```diff
@@ -71,7 +71,7 @@ export class NodePostgresDriverDialect implements DriverDialect {
     }
   }
 
-  setQueryTimeout(props: Map<string, any>, sql?: any, wrapperQueryTimeout?: any) {
+  setQueryTimeout(props: Map<string, any>, wrapperQueryTimeout?: any) {
```

Comment:
Need to revert this change since we are going with the new fix that does not require changing the interface